src/OrchardCore/OrchardCore.Queries.Abstractions/LuceneQueryResults.cs

```diff
@@ -1,7 +1,10 @@
+using System;
 using System.Collections.Generic;
+using OrchardCore.Queries;
 
 namespace OrchardCore.Queries
 {
+    [Obsolete("Moved to OrchardCore.Lucene.Abstractions", false)]
```

Comment:
This class has been deprecated and we will be removed in the major release, please use OrchardCore.Lucene.Abstractions instead.